compound_disease/compound_treats_disease/edge_prediction_experiment/edge_prediction_experiment.py

```diff
@@ -283,31 +318,111 @@
 edges_added_df.head(10)
 
 
-# In[17]:
+# In[20]:
 
 
 ax = sns.scatterplot(x="precision", y="edges", hue="in_hetionet", data=edges_added_df)
 ax.set(yscale="log")
 
 
-# In[18]:
+# In[21]:
 
 
 edges_added_df.to_csv("output/precision_ctd_edges_added.tsv", index=False, sep="\t")
 
 
 # # Total Recalled Edges
 
-# How many edges of hetionet can we recall using a cutoff score of 0.5?
+# How many edges of hetionet can we recall using an equal error rate cutoff score?
```

Comment:
This section takes a different view at finding the optimal cutoff. Instead of use precision-recall curves I used AUROC curves instead. In this case equal error rate means I want a ROC cutoff point that gives me the same number of false positives as false negatives (equal error). I was exploring another route to call Hetionet edges.